api/src/start.ts

```diff
@@ -1,9 +1,25 @@
 import app from './Server';
 import { config } from './config';
 import { logger } from './logger';
+import { getDbClientInstance } from './database/dbClient';
+import { createDatabase } from './database/createDatabase';
 
-// Start the server
-const port = config.serverPort;
-app.listen(port, () => {
-    logger.info('Express server started on port: ' + port);
-});
+export const runServer = async () => {
+    // Create database
+    await createDatabase();
+    // Database client
+    const dbClient = getDbClientInstance();
+
+    // Start the server
+    const port = config.serverPort;
+    await new Promise((resolve) => {
+        app.listen(port, () => {
```

Comment:
It's been a long time since I've done node.js work but I didn't think the `await new Promise` hack here was necessary. I think what you're doing here is creating a promise that'll never fulfill so that this main method waits forever thus keeping the app from ending. But I thought one or more of the following was true:

* Node won't exit as long as there is work pending so just the `listen` is OK
* `listen` takes an argument that does this blocking hack for you

Reply:
I removed the `await new Promise` because there is no need for it yet.